staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/errors.go

```diff
@@ -44,7 +44,6 @@ func BadGatewayError(w http.ResponseWriter, req *http.Request) {
 func Forbidden(ctx request.Context, attributes authorizer.Attributes, w http.ResponseWriter, req *http.Request, reason string, s runtime.NegotiatedSerializer) {
 	msg := sanitizer.Replace(forbiddenMessage(attributes))
 	w.Header().Set("X-Content-Type-Options", "nosniff")
-	w.WriteHeader(http.StatusForbidden)
```

Comment:
https://golang.org/pkg/net/http/#ResponseWriter

The doc:
        // Changing the header map after a call to WriteHeader (or
        // Write) has no effect unless the modified headers are
        // trailers.